Documentation/common-workflow.md

```diff
@@ -0,0 +1,6 @@
+# Common Workflow
```

Comment:
Suggest rm Documentation/common-workflow.md after inserting this same info in Documentation/getting-started as per my comment on that file, graf 2, below.